### ISSUE TITLE
Entered Pool selection missing with different coins

### ISSUE THREAD

**Issue by vRobM** [BobbyGR/AIOMiner#98]
Pool / Coin Settings
Pick a coin, add a pool, save
Now pick another coin that can be mined by the same pool or using same algo
the pool list is empty ;-/

It should have saved the pool list globally for all coins. User knows which to pick. ;-)
Don't force user to re-add the same pool, over and over again. ;-/

**Comment by BobbyGR** [comment]
How do I know what coins a pool supports?

**Comment by Caparino** [comment]
You can't automatically pull the global pool list from a single port. I'm guessing you're new to mining?

**Comment by vRobM** [comment]
We're not on the same page yet. 
Let me rephrase.

If I enter a custom pool for a cryptonight coin, and I want to add another coin to mine, which also uses cryptonight, why isn't my custom, already entered pool available in the pool list for the second coin?

Entering a custom pool should be visible on any other coin I select, regardless if it has a known default pool or not. The custom pool settings should also all be available, cloned from the first.

**Comment by BobbyGR** [comment]
So you want all of the pool you put in to be selectable regardless of the coin?

**Comment by vRobM** [comment]
Yes, otherwise it has to be re-entered manually for every single one.
If it persists, it's just a selection. And if made default, it shouldn't need to be selected at all. It would be default.

**Comment by BobbyGR** [comment]
Ok, so let's say I add 10 pools, when I make coin X, when I got select a pool I could sift through those pools I added then select one of them and just change the port.  I get it.  Ill look into this option.  With the 9 release it will be a crowdfunded pool list, meaning if you add a pool, you are adding that option for everyone.  Then as people use that pool, the rnknof that pool will adjust.  So, the list will grow per coin, need to see how could put in this with that.
